ui/urls.py

```diff
@@ -8,13 +7,6 @@
 
 urlpatterns = [
     url(
-        r"^robots\.txt$",
```

Comment:
Why delete?

Reply:
robots.txt does not exist in this repo.

adding useful a robots.txt would require a sitemap.xml - overall adding useful robots.txt will take a few hours and I see no reason to block this PR for that feature.

Will add a ticket.